README.md

```diff
@@ -8,141 +8,337 @@
 tools {
     jfrog 'jfrog-cli'
 }
 ```
 
+<details>
+  <summary>Scripted Pipeline</summary>
+
+```groovy
+withEnv(["JFROG_BINARY_PATH=${tool 'jfrog-cli'}"]) {
+    // The 'jf' tool is available in this scope.
+}
+```
+
+</details>
+
 **Step 2:**
 Use the **jf** step to execute any JFrog CLI command as follows:
+
 ```groovy
 // Upload all files in the target directory to the my-repo Artifactory repository.
 jf 'rt u target/ my-repo/'
 ```
-> **_IMPORTANT:_** Notice the single quotes wrapping the command right after the **jf** step definition.  
 
-### Setting the build name and build number
+> **_IMPORTANT:_** Notice the single quotes wrapping the command right after the **jf** step definition.
 
-The plugin automatically sets the following environment variables: *JFROG_CLI_BUILD_NAME* and *JFROG_CLI_BUILD_NUMBER* with Jenkins's job name and build number respectively. 
+### Setting the Build Name and the Build Number
+
+The plugin automatically sets the following environment variables: _JFROG_CLI_BUILD_NAME_ and _JFROG_CLI_BUILD_NUMBER_
+with Jenkins's job name and build number respectively.
 You therefore don't need to specify the build name and build number on any of the build related JFrog CLI commands.
-If you wish to change the default values, add the following code to your pipeline script: 
+If you wish to change the default values, add the following code to your pipeline script:
+
 ```groovy
 environment {
     JFROG_CLI_BUILD_NAME = "my-build-name"
     JFROG_CLI_BUILD_NUMBER = "18"
 }
 ```
 
-**Example**
+### Using Multiple JFrog Platform Instances
+
+If you have multiple JFrog Platform instances configured, you can use the `–-server-id` command option with
+the server ID you configured for the instance. For example:
+
+```groovy
+jf 'rt u test-file my-repo –-server-id server-1'
+jf 'rt u test-file my-repo –-server-id server-2'
+```
+
+## Examples
+
+<details>
+  <summary>Uploading a File</summary>
+
 ```groovy
 pipeline {
     agent any
     tools {
         jfrog 'jfrog-cli'
     }
     stages {
-        stage ('Testing') {
+        stage('Testing') {
             steps {
                 // Show the installed version of JFrog CLI.
                 jf '-v'
-                
+
                 // Show the configured JFrog Platform instances.
                 jf 'c show'
-                
+
                 // Ping Artifactory.
                 jf 'rt ping'
-                
+
                 // Create a file and upload it to a repository named 'my-repo' in Artifactory
                 sh 'touch test-file'
                 jf 'rt u test-file my-repo/'
-                
+
                 // Publish the build-info to Artifactory.
                 jf 'rt bp'
             }
         }
     }
 }
 ```
-> **_NOTE:_** If you have multiple JFrog Platform instances configured, you can use the –server-id command option with the server ID you configured for the instance. For example:
+
+</details>
+
+<details>
+  <summary>Maven Example</summary>
```

Comment:
Maven Example --> Maven

(Let's remove the `Example` suffix from all the examples titles).